cumulusci/tasks/release_notes/parser.py

```diff
@@ -283,3 +284,35 @@ def _add_issue_comment(self, issue):
                 break
         if not has_comment:
             issue.create_comment("{} {}".format(comment_prefix, version_str))
+
+
+class InstallLinkParser(BaseChangeNotesParser):
+    prod_url_template = (
+        "https://login.salesforce.com/packaging/installPackage.apexp?p0={}"
+    )
+
+    test_url_template = (
+        "https://test.salesforce.com/packaging/installPackage.apexp?p0={}"
+    )
```

Comment:
We should use PROD_LOGIN_URL and SANDBOX_LOGIN_URL from cumulusci.oauth.salesforce, and then the URLs will be generated correctly for non-prod environments too.